apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -173,9 +171,7 @@ var TopInstructions = React.createClass({
         height: this.props.height - RESIZER_HEIGHT
       },
       this.props.noVisualization && styles.noViz,
-      this.props.isEmbedView && Object.assign({}, styles.embedView, {
-        left: this.props.embedViewLeftOffset
-      })
+      this.props.isEmbedView && styles.embedView,
     ];
```

Comment:
As of this change, both implementations of top instructions are responsive. If this changes in the future, pageConstants.isResponsive should be used to determine whether each top instructions will be responsive.

AFAIK there were no instances of TopInstructionsCSF on embed levels. If you can think of one please let me know.